app/assets/stylesheets/cypress/_products.scss

```diff
@@ -0,0 +1,53 @@
+// Place all the styles related to the products controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+input[type=radio]:checked + label {
+  border-bottom: 4px solid $brand-primary;
+}
+
+input[type=checkbox]:checked + label {
+  border-left: 4px solid $brand-primary;
+}
+
+input[type=checkbox] + label {
+  text-align: left;
+  margin-bottom: 1em;
+}
+
+.cannot_select, .cannot_select:hover, .cannot_select:active {
+  background: $gray-lighter;
+  cursor: not-allowed;
+}
+
+.form-group:last-child {
+  margin-top: 1em;
+}
+
+.col-sm-1 > button, .col-sm-2 > button,
+.col-sm-3 > button, .col-sm-4 > button,
+.col-sm-1 > label, .col-sm-2 > label,
+.col-sm-3 > label, .col-sm-4 > label {
+  width: 100%;
+}
```

Comment:
It might be better to just adjust the markup to include the `.btn-block` class??